### DISCUSSION TITLE
Partial Loss of Styles for Rounded Buttons and Dropdowns

### DISCUSSION THREAD

**Discussion by A330454** [primefaces/community#773] [PrimeNG] [answered]
Hello Community,

I recently encountered with partial loss of styles specifically affecting rounded buttons and dropdowns. Strangely, all other components and styles remain intact and function as expected.

Components Affected: Rounded buttons and dropdowns only.
Symptoms: These components have lost their styling - particularly, the rounded corners and any associated styling specific to these components are not being rendered.
Other Components: All other components in the library appear unaffected and are displaying their styles correctly.
Functionality: Despite the styling issues, the functionality of these components remains unaltered.

PrimeNG and Angular Versions: I am using PrimeNG version [16.0.2] and Angular version [16.1.0].
Browser Console: No relevant errors or warnings in the browser console that point towards this issue.

I would appreciate any insights or guidance on the following:

- Potential Causes: What could be causing this selective loss of styles post-update?
- Debugging Tips: Recommended steps to identify and rectify the issue.
- Similar Experiences: Has anyone else faced a similar issue, and how was it resolved?

Screenshots: 

Button:
![image](https://github.com/primefaces/community/assets/95514323/21506985-1a85-4ddd-9007-272967a0efc0)

![image](https://github.com/primefaces/community/assets/95514323/e51a57a8-a8ac-4a2b-9b00-488a3dc14549)

![image](https://github.com/primefaces/community/assets/95514323/863969b1-3e78-4bf6-b5df-e5c6efd1ddf8)

![image](https://github.com/primefaces/community/assets/95514323/d82adb10-9158-46c7-be26-35cfb094192b)

![image](https://github.com/primefaces/community/assets/95514323/272df7ec-a10b-4c8e-9265-d7b77347e349)

I am eager to resolve this issue and would greatly value any input from the community. Thank you in advance for your time and assistance.

Best regards,
Jacobo

**Comment by A330454** [comment]
![image](https://github.com/primefaces/community/assets/95514323/20f46d1b-734b-4a15-acd1-8b1d631bd946)

I had tried updating and downgrading primeng dependencies, but nothing has worked for me.

**Comment by cetincakiroglu** [comment]
Hi,

Could you please share a stackblitz reproducer so we can identify the issue clearly?

**Reply by A330454** [reply]
Of course! Thanks for the help.

https://stackblitz.com/~/github.com/A330454/StyleFailureDemo

**Reply by cetincakiroglu** [reply, marked as answer]
Hi,

You're overriding the PrimeNG styles by bootstrap.

Do the things below and the PrimeNG styles will be applied or manage CSS loading sequence so as not to override each other.
1-) Remove bootstrap cdn from the index.html
2-) Add the lines below to the angular.json styles array,

```
            "styles": [
              "node_modules/primeng/resources/themes/lara-light-blue/theme.css",
              "node_modules/primeng/resources/primeng.min.css",
              "src/styles.scss"
            ],
```
You can also do it by adding styles above inside of the src/styles.scss. The result will be the same but you need to manage bootstrap CSS which overrides the PrimeNG styles.

**Reply by A330454** [reply]
Thank you @cetincakiroglu ,

I removed boostrap and delet the lines of code from anglar.json, and index.html, that let to get back the styles I wanted.

```json
"styles": [
-"node_modules/bootstrap/dist/css/bootstrap.min.css",
"node_modules/primeng/resources/themes/lara-light-blue/theme.css",
"node_modules/primeng/resources/primeng.min.css",
"src/styles.scss"
],
"scripts": [
-"node_modules/bootstrap/dist/js/bootstrap.min.js"]
```

and uninstalling boostrap from my project with 

```
npm uninstall bootstrap --legacy-peer-deps
```
